src/views/About/About.tsx

```diff
@@ -1,19 +1,12 @@
 import { Trans } from 'react-i18next';
 import { i18n } from '~core/localization';
-import config from '~core/app_config';
-import history from '~core/history';
-import { APP_ROUTES } from '~core/app_config/appRoutes';
 import s from './About.module.css';
 
-function gotoMap() {
-  history.push(APP_ROUTES.map);
+export function AboutPage({ toHomePage }: { toHomePage: () => void }) {
```

Comment:
do we really need AboutPage ? maybe lets remove it and rename AboutText -> AboutPage?

Reply:
agree